includes/environment.inc

```diff
@@ -192,28 +192,13 @@ function drush_bit_bucket() {
  *   MINGW* (e.g. MINGW32)
  */
 function _drush_get_os($os = NULL) {
-  // The special os "CWRSYNC" can be used to indicate that we are testing
-  // a path that will be passed as an argument to cwRsync, which requires
-  // that the path be converted to /cygdrive/c/path, even on DOS or Powershell.
-  // The special os "RSYNC" can be used to indicate that we want to assume
-  // "CWRSYNC" when cwrsync is installed, or default to the local OS otherwise.
-  if (strtoupper($os) == "RSYNC") {
-    $os = _drush_get_os("LOCAL");
-    // For now we assume that cwrsync is always installed on Windows, and never installed son any other platform.
-    return drush_is_windows($os) ? "CWRSYNC" : $os;
-  }
   // We allow "LOCAL" to document, in instances where some parameters are being escaped
   // for use on a remote machine, that one particular parameter will always be used on
   // the local machine (c.f. drush_backend_invoke).
   if (isset($os) && ($os != "LOCAL")) {
```

Comment:
If we are going to keep `_drush_get_os`, I'd prefer to keep MINGW and delete LOCAL. At this point, though, I think we should just replace all calls to `_drush_get_os()` to PHP_OS. Doing this requires removing LOCAL from backend.inc though.

Reply:
I think the abstraction can still be important enough to justify keeping `_drush_get_os();`.

I just pushed an update that removed everything but the LOCAL check and the return of PHP_OS.  I'll add the MINGW code back in.